src/card/Card.js

```diff
@@ -0,0 +1,208 @@
+import PropTypes from 'prop-types';
```

Comment:
There is a duplicate file in the containers, if this is being moved than let's delete the container folder